src/main/java/frc/trigon/robot/poseestimation/poseestimator/PoseEstimator6328.java

```diff
@@ -89,6 +90,8 @@ public void addOdometryObservation(OdometryObservation observation) {
     }
 
     public void addVisionObservation(VisionObservation observation) {
+//        if (DriverStation.isAutonomous() || DriverStation.isDisabled())
+//            return;
```

Comment:
Remove. Also maybe implement an easy way to change between using cameras and odometry for autonomous